bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java

```diff
@@ -584,15 +584,6 @@ public void testWithDiskFullReadOnlyDisabledOrForceGCAllowDisabled() throws Exce
         } catch (NoWritableLedgerDirException e) {
             // expected
         }
-
```

Comment:
in the existing implementation, when Bookie is initialized, it will eventually call EntryLogger.initialize() -> EntryLogger.createNewLog() and it would fail with exception because of https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsManager.java#L184 . But now with this change we are not creating newlog during initialization of Bookie ( and EntryLogger), EntryLog would be created only after receiving first EntryLogger.addEntry call, hence Bookie initialization would succeed in this case though IsForceGCAllowWhenNoSpace is set to false.